src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: New Relic Service Broker for VMware Tanzu
+tags:
+  - Integrations
+  - Cloudfoundry
+  - Tanzu
+metaDescription: Use our integration to gain increased visibility into the performance of your VMware Tanzu environment.
+freshneessValidatedDate: never 
+---
+
+import infrastructureVmwareTanzuImport from 'images/vmware-service-broker-import.png'
```

Comment:
I'm not convinced that these screenshots are entirely necessary. If we could get a full screenshot of each page that these are taken from, we might be able to find a better, shorter way to capture the instructions these correspond to.

These are also .png and will need to be converted to webp